tests/test_app_text.py

```diff
@@ -110,17 +110,17 @@ def test_asset_variable_replacement(self):
                                        asset=test_asset,
                                        variables=test_vars)
         rf_id = int(resource.asset.split()[-1])
-        self.assertEqual(rf_id, TEST_USER_ID)
+        assert rf_id == TEST_USER_ID
 
         invalid_asset = "Not a real {variable}!"
         resource = UnversionedResource(test_url,
                                        asset=invalid_asset,
                                        variables=test_vars)
         error_key = resource.asset.split()[-1]
         if sys.version_info[0] < 3:
```

Comment:
Is this version-dependent check still required even with the `unicode_literals` import?

Reply:
It looks like it is

Reply:
Pulled this branch, and see why this check is needed:
```
>>> p asset
u'Missing asset variable u\\'variable\\''
```
That hardcoded `u\\` shouldn't be in there, me thinks...

Reply:
Agreed, just found where it's being added. Removing it...